myuw/templates/handlebars/card/accounts/account_medicine.html

```diff
@@ -24,7 +24,7 @@ <h4 class="pw-exp-date">Password expiration</h4>
             	<span class="card-badge-action pull-right {{#if expires_in_30_days_or_less}}password-alert{{/if}}">in {{days_before_med_pw_expires}} days* </span>
 	    </div><!-- .pull-right -->
             <div><p class="update-note">*Expiration date gets updated nightly.</p></div>
-            <div class="pw-link"><a href="https://services.uwmedicine.org/passwordportal/login.htm">Change UW Medicine password</a></div>
+            <div class="pw-link links-container"><a href="https://services.uwmedicine.org/passwordportal/login.htm">Change UW Medicine password</a></div>
```

Comment:
You can attach the style you need to pw-link class rather than create links-container since they both uniquely target the same element.